arduino/Norbot_v2.0_arduino/Norbot_v2.0_arduino.ino

```diff
@@ -0,0 +1,120 @@
+
```

Comment:
Can we call this something more consistent with the rest of the naming convention i.e. ``arduino/i2c_slave_code_norbot2/i2c_slave_code_norbot2.ino`` or something

Reply:
On a similar note, I feel like Norbot2 isn't the most descriptive name. How about ``RCCarRobot`` or something?